Description: enable integration tests
 Upstream disabled the generation of dedicated makefile to run the integration tests
 in configure.ac. This patch enables the output of functional-system-tests.mk.
Author: Joël Krähemann <jkraehemann@gmail.com>
Origin: upstream
Forwarded: not-needed
Last-Update: 2017-11-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/configure.ac
+++ b/configure.ac
@@ -272,6 +272,7 @@
 
 AM_EXTRA_RECURSIVE_TARGETS([ags-docs])
 AC_CONFIG_FILES([
+functional-system-tests.mk
 Makefile
 docs/reference/libags/Makefile
 docs/reference/libags-audio/Makefile
